server/trillian_log_signer/main.go

```diff
@@ -59,10 +65,24 @@ func main() {
 	}
 	defer db.Close()
 
+	ctx, cancel := context.WithCancel(context.Background())
+	go util.AwaitSignal(cancel)
+
+	hostname, _ := os.Hostname()
+	instanceID := fmt.Sprintf("%s.%d", hostname, os.Getpid())
```

Comment:
Maybe later: "This package is currently in development and the API may not be stable."